scripts/newService.js

```diff
@@ -73,41 +73,50 @@ export interface ${state.model}UpdateVariables {}
 export type ${state.model}UpdateResponse = Object;`;
 }
 
-function createIndexFile(){
-  fs.writeFileSync(`${state.directory}/index.ts`, state.indexFileContent)
+function createIndexFile() {
+  fs.writeFileSync(`${state.directory}/index.ts`, state.indexFileContent);
 }
-function createServiceFile(){
-  fs.writeFileSync(`${state.directory}/${state.serviceName}.service.ts`, state.serviceFileContent)
+function createServiceFile() {
+  fs.writeFileSync(
+    `${state.directory}/${state.serviceName}.service.ts`,
+    state.serviceFileContent,
+  );
 }
-function createQueriesFile(){
-  fs.writeFileSync(`${state.directory}/${state.serviceName}.queries.ts`, state.queriesFileContent)
+function createQueriesFile() {
+  fs.writeFileSync(
+    `${state.directory}/${state.serviceName}.queries.ts`,
+    state.queriesFileContent,
+  );
 }
-function createTypesFile(){
-  fs.writeFileSync(`${state.directory}/${state.serviceName}.types.ts`, state.typesFileContent)
+function createTypesFile() {
+  fs.writeFileSync(
+    `${state.directory}/${state.serviceName}.types.ts`,
+    state.typesFileContent,
+  );
 }
 
 const init = async () => {
   // const questions = setQuestions();
   // const responses = await inquirer.prompt(questions);
-  
+
   // setDirectoryName(responses.serviceName);
```

Comment:
@luizfelipelaviola Why did we remove the inquirer functionality? Or was this Maddy?